### PR TITLE
Compile against bokeh 2.3

### PR DESCRIPTION
**Pull request by philippjfr**

### CODE REVIEW

File: panel/models/card.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export class CardView extends ColumnView {
 
     const {background, button_css_classes, header_color, header_tag, header_css_classes} = this.model
 
-    this.el.style.backgroundColor = background != null ? background : ""
+    this.el.style.backgroundColor = (background != null ? background : "" as any)
```
Comment (mattpap) [inline panel/models/card.ts:54]: Use `color2css()` from `core/util/color`.

Comment (mattpap) [reply panel/models/card.ts:54]: In general one can't assume anymore that a color is a string, or even if it's actually a string, that it's a CSS color spec or is a widely supported CSS color spec (e.g. we allow `#RRGGBBAA` syntax, which isn't widely supported yet).

Comment (philippjfr) [reply panel/models/card.ts:54]: Thanks!

**Comment by xavArtley** [comment]
Hello, I wanted to try this PR however when I try to compile I have this error:
```
Compiling TypeScript (39 files)
panel/models/file_download.ts:3:26 - error TS2306: File 'C:/Users/xa235208/Documents/Developments/modules/panel/panel/node_modules/@bokeh/bokehjs/build/js/types/styles/buttons.css.d.ts' is not a module.

3 import * as buttons from "@bokehjs/styles/buttons.css"
                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
panel/models/reactive_html.ts:38:17 - error TS2345: Argument of type 'string' is not assignable to parameter of type 'number'.

38   return !isNaN(str) && !isNaN(parseFloat(str))
                   ~~~
panel/models/singleselect.ts:6:25 - error TS2306: File 'C:/Users/xa235208/Documents/Developments/modules/panel/panel/node_modules/@bokeh/bokehjs/build/js/types/styles/widgets/inputs.css.d.ts' is not a 
module.

6 import * as inputs from "@bokehjs/styles/widgets/inputs.css"
```

My bokeh version is this one:
(py38) C:\Users\xa235208\Documents\Developments\modules\panel>pip show bokeh
Name: bokeh
Version: 2.3.0.dev9
Summary: Interactive plots and applications in the browser from Python
Home-page: http://github.com/bokeh/bokeh
Author: Bokeh Team
Author-email: info@bokeh.org
License: BSD-3-Clause
Location: c:\users\xa235208\documents\python\pyenv\py38\lib\site-packages\bokeh-2.3.0.dev9-py3.8.egg
Requires: PyYAML, python-dateutil, Jinja2, numpy, pillow, packaging, tornado, typing-extensions
Required-by: hvplot, datashader, panel

**Comment by philippjfr** [comment]
@xavArtley I get those too, ignore them for now (they do not break anything). @mattpap Any ideas about those CSS errors?

**Comment by mattpap** [comment]
> Any ideas about those CSS errors?

Not really. I would `git clean -dfx` and rebuild from scratch. There's a possibility that build caching is the culprit.

**Comment by philippjfr** [comment]
>Not really. I would git clean -dfx and rebuild from scratch. There's a possibility that build caching is the culprit.

It's definitely not as far as I can tell. I've removed the `dist` and `node_modules` directories a bunch of times and am still seeing this.

**Comment by mattpap** [comment]
I see, CSS support in extensions was added after dev9. I will push out dev10 soon. In the meantime the best option is to use locally built bokehjs.

**Comment by mattpap** [comment]
@philippjfr, I published 2.3.0-dev10.

**Comment by philippjfr** [comment]
@mattpap Thanks, but sadly I'm still seeing those errors. Is there anything else to configure?

**Comment by mattpap** [comment]
I fixed the underlying issue in PR https://github.com/bokeh/bokeh/pull/10838. If you want to test this, then issue `npm pack` in `bokeh/bokehjs` and then use `file:~/bokeh/bokehjs/bokeh-bokehjs-2.3.0-dev.10.tgz` (or similar) in panel's `package.json` (instead of `^2.3.0-dev.10`).

**Comment by mattpap** [comment]
There is one more issue:
```ts
models/file_download.ts:174:44 - error TS2339: Property 'btn_type' does not exist on type 'typeof import("/home/mateusz/repo/panel/panel/node_modules/@bokeh/bokehjs/build/js/types/styles/buttons.css")'.

174       this.anchor_el.classList.add(buttons.btn_type(this.model.button_type))
```
with can be fixed by replacing `btn_type()` with:
```ts
import * as buttons from "styles/buttons.css"
buttons[`btn_${this.model.button_type}` as const]
```

**Comment by philippjfr** [comment]
Thanks, really appreciate it!

**Comment by philippjfr** [comment]
@mattpap With Bokeh 2.3.0dev11:

```
panel/models/file_download.ts:174:36 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'typeof import("/Users/philippjfr/development/pyviz_panels/panel/node_modules/@bokeh/bokehjs/build/js/types/styles/buttons.css")'.
  No index signature with a parameter of type 'string' was found on type 'typeof import("/Users/philippjfr/development/pyviz_panels/panel/node_modules/@bokeh/bokehjs/build/js/types/styles/buttons.css")'.

174       this.anchor_el.classList.add(buttons[`btn_${this.model.button_type}` as const])
```

**Comment by philippjfr** [comment]
Nevermind my fault.

**Comment by JackDapid** [comment]
> @xavArtley I get those too, ignore them for now (they do not break anything). @mattpap Any ideas about those CSS errors?

Not really sure if it related, but with: 
Bokeh-2.3.0
Panel-0.11.1
Holoviews-1.14.2
geoviews-1.9.1
All is fine on the server side, but on client slide console bokehjs:
`Error: Cannot find module '@bokehjs/styles/buttons'`

while all fine with bokeh-2.2.3 , Panel-0.10.3,  Holoviews-1.14.1, geoviews-1.8.2

**Pull request merged**